app/mailers/alert_mailer.rb

```diff
@@ -5,7 +5,7 @@ class AlertMailer < ApplicationMailer
   def alert(subscription_id, vacancy_ids)
     subscription = Subscription.find(subscription_id)
     @subscription = SubscriptionPresenter.new(subscription)
-    vacancies = Vacancy.where(id: vacancy_ids).order(:expires_on).order(:expiry_time)
+    vacancies = Vacancy.where(id: vacancy_ids).order(:expires_on).order(:expires_at)
```

Comment:
Do we need to chain the two attributes, or could we just order by `:expires_at`?